conf/tables.conf

```diff
@@ -103,6 +104,17 @@ readonly SCHEMA_RESULTS=$(cat <<EOF
 ]
 EOF
 )
+readonly UPDATE_WHERE_PUBLISHED=$(cat <<EOF
+  UPDATE \`${BQ_PUBLIC_DATASET}.${BQ_METADATA_TABLE}\`
+  SET where_published =
+    CASE
+      WHEN '${BQ_PUBLIC_DATASET}.${BQ_PUBLIC_TABLE}' NOT IN UNNEST(where_published)
+        THEN ARRAY_CONCAT(where_published, ['${BQ_PUBLIC_DATASET}.${BQ_PUBLIC_TABLE}'])
```

Comment:
Although we are not likely at all to delete tables, it is possible that we will delete a table and re-create a new one with the same name.  In that case, although the table name is the exact same string, its URL will be different.  So, I think we need to save the URL here.